Makefile

```diff
@@ -67,7 +67,7 @@ dev-deps:
 dev-switch:
 	opam update
 # Ensuring that either a dev switch already exists or a new one is created
-	[[ $(shell opam switch show) == $(shell pwd) ]] || \
+	test "$(shell opam switch show) == $(shell pwd)" || \
```

Comment:
Seems strange to quote the whole comparison term. Shouldn't you be quoting each one of the operands?

Reply:
Also, if you worry about `sh` compatibility, I guess it is better to use `=` rather than `==` (they are synonyms under `bash`).

Reply:
Agreed. I'll update asap.